### DISCUSSION TITLE
Shortcut for delayed capture?

### DISCUSSION THREAD

**Discussion by joaociocca** [flameshot-org/flameshot#3100] [Ideas]
Because of Windows, and the printscreen key being so far off on keyboards, I got used to win+shift+s for capturing screens... I currently change Flameshot's capture to it using Power Toys on Windows and the configuration options on Kubuntu. But it would be really nice if I could actually assign the simple capture to it, and free up PrtSc key to use a fixed-number, like 2 or 3 seconds, for delayed capture.

**Comment by mmahmoudian** [comment]
On Windows we do not yet support CLI (follow #2118), but on Linux and macOS you can use `--delay <milliseconds>` . For example `flameshot gui --delay 2000`
